src/AltinnCore/AltinnCore.Runtime.Db/Models/FormData.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AltinnCore.Runtime.Db.Models
+{
+    public class FormData
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "reporteeId")]
+        public string ReporteeId { get; set; }
+
+        [JsonProperty(PropertyName = "reporteeElementId")]
+        public string ReporteeElementId { get; set; }
+
+        [JsonProperty(PropertyName = "serviceId")]
+        public string ServiceId { get; set; }
+
+        [JsonProperty(PropertyName = "serviceEdition")]
+        public string ServiceEdition { get; set; }
```

Comment:
Will be unnecessary after change removing editions is completed

Reply:
Removed